### DISCUSSION TITLE
Removing player tag from title

### DISCUSSION THREAD

**Discussion by roxaos** [streamlink/streamlink#6046] [Q&A] [answered]
### Description

![mpvtitle](https://github.com/streamlink/streamlink/assets/47874327/b8f47788-b0d0-4687-a5a9-4ac690ebdec9)

I'm attempting to remove the "- mpv" text after the title/author text. I haven't quite found a way to do so in the documentation. Unsure if this is better directed at the mpv repo, but figured I'd ask.

**Comment by bastimeyer** [comment, marked as answer]
This is completely unrelated to Streamlink. You can't expect any help with such a low-effort question. You didn't even mention what we're looking at here.

This application / widget most likely simply appends the process name to each process-window title it lists, so you have to consult its documentation / manual / source-code or ask its developers instead.

The player title is set by Streamlink via the `--title` (template string) and `--player-args` CLI arguments, which you can also check in the debug log output. The resulting player launch arguments depend on the player that's detected by Streamlink. In case of mpv, it sets mpv's `--force-media-title` launch argument. Streamlink itself does not append ` - mpv`.

- https://streamlink.github.io/cli.html#cmdoption-title
- https://streamlink.github.io/cli.html#cmdoption-player-args
- https://streamlink.github.io/cli.html#cmdoption-loglevel
- https://mpv.io/manual/stable/#options-force-media-title
